x-pack/plugins/security_solution/common/api/timeline/resolve_timeline/resolve_timeline_route.schema.yaml

```diff
@@ -34,17 +36,15 @@ paths:
             application/json:
               schema:
                 type: object
+                required: [data]
                 properties:
                   data:
                     type: object
                     properties:
                       getOneTimeline:
-                        oneOf:
-                          - $ref: '../model/components.yaml#/components/schemas/TimelineResponse'
-                          - nullable: true
-                required:
-                  - data
+                        allOf:
```

Comment:
@michaelolo24 In your original PR, there's a `nullable: true` here but that didn't compile. Wdyt about leaving this out for now?

Reply:
Yea, we can leave it out for now and take a look at it afterwards? Thanks!

Reply:
@janmonschke to make it working with code generation you need to omit `allOf` and place `nullable` next to `$ref` like below

```yaml
properties:
  getOneTimeline:
    $ref: '../model/components.schema.yaml#/components/schemas/TimelineResponse'
    nullable: true
```

It should generated an expected Zod schema. The only problem with it's not a valid OpenAPI `3.0` syntax but it's ok for `3.1`.

Reply:
Makes sense! Thanks @maximpn ! https://github.com/elastic/kibana/pull/188844/commits/73d7ab04f6d8239faa6bae2d35bc1e224796de99

Reply:
@janmonschke I think a lot of the missing nullable fields are because of this. Not sure if you want to add them from my PR or change the syntax to match what @maximpn mentioned here...